ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/transaction/Rejections.scala

```diff
@@ -51,9 +53,7 @@ private[transaction] class Rejections(metrics: Metrics) {
   def buildRejectionEntry(
```

Comment:
Maybe let's rename this to `buildPotentialRejectionEntry` or something to make it clear we may not use the result?

Reply:
Problem is, then the fact that reject builds a "potential" rejection may become confusing. I'm tempted to make it `private` and wrap it as `buildPostExecutionRejectionEntry` to be used in the ledger time validator.

Reply:
That sounds like a good plan to me.

Reply:
2f5068e

Reply:
And 94a7787; also, I think 1df697f is more correct for the post-executor.